src/components/ManageAccountsDrawer/ManageAccountsDrawer.tsx

```diff
@@ -1,42 +1,65 @@
-import { ethChainId } from '@shapeshiftoss/caip'
-import { useEffect, useMemo, useState } from 'react'
+import type { ChainId } from '@shapeshiftoss/caip'
+import { useCallback, useMemo, useState } from 'react'
 import { assertUnreachable } from 'lib/utils'
 
 import { DrawerWrapper } from './components/DrawerWrapper'
 import { ImportAccounts } from './components/ImportAccounts'
+import { SelectChain } from './components/SelectChain'
 
 export type ManageAccountsDrawerProps = {
   isOpen: boolean
   onClose: () => void
 }
 
-type ManageAccountsStep = 'manageAccounts' | 'selectChain' | 'ledgerOpenApp' | 'importAccounts'
+type ManageAccountsStep = 'selectChain' | 'ledgerOpenApp' | 'importAccounts'
 
 export const ManageAccountsDrawer = ({ isOpen, onClose }: ManageAccountsDrawerProps) => {
-  const [step, setStep] = useState<ManageAccountsStep>('manageAccounts')
+  const [step, setStep] = useState<ManageAccountsStep>('selectChain')
+  const [selectedChainId, setSelectedChainId] = useState<ChainId | null>(null)
 
-  // TEMP: Set the initial step to 'importAccounts' until we have the other steps implemented
-  useEffect(() => {
-    setStep('importAccounts')
-  }, [])
+  const isLedger = false
+
+  const handleNext = useCallback(() => {
+    switch (step) {
+      case 'selectChain':
+        if (isLedger) {
+          setStep('ledgerOpenApp')
+        }
+        setStep('importAccounts')
+        break
+      case 'ledgerOpenApp':
+        setStep('importAccounts')
+        break
+      case 'importAccounts':
+        onClose()
+        break
+      default:
+        assertUnreachable(step)
+    }
+  }, [isLedger, onClose, step])
+
+  const handleSelectChainId = useCallback(
+    (chainId: ChainId) => {
+      setSelectedChainId(chainId)
+      handleNext()
```

Comment:
This is borked and doesn't close the drawer currently - also should probably call `handleClose()` if/when added, so we can reset the step to the initial one here as well

Reply:
Disregard me, this is effectively called on next click (any next click, not only on done click) so no route reset, although it being borked when clicking "Done" and not closing the drawer still stands